### PR TITLE
add tests to ensure generators give correct output for issue 571

### PR DESCRIPTION
**Pull request by turbomam**
#571 

I wrote some code to generate YAML, SQL and Java programmatically. They're all OK.

Now I have to try ProjectGenerator

### CODE REVIEW

File: tests/test_issues/test_issue_571_sql_java.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    # def test_valid_math(self):
+    #     self.assertEqual(1, 2 / 2)
+
+    # def test_bogus_math(self):
+    #     self.assertEqual(1, 0.5 + 0.501)
+
+    def test_sql_ddl_gen(self):
+        # todo make the schema creation a fixture and split out the different generations
+        schema_id = 'http://example.com/issue_571_schema'
+        issue_571_schema = SchemaDefinition(name="issue_571_schema", id=schema_id)
```
Comment (cmungall) [inline tests/test_issues/test_issue_571_sql_java.py:31]: a trifle wordy but YMMV!

Comment (turbomam) [reply tests/test_issues/test_issue_571_sql_java.py:36]: > a trifle wordy but YMMV!

I'm trying to figure how to use the kitchen sink schema as input

File: tests/test_issues/test_issue_571_sql_java.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        pgen = ProjectGenerator()
+        pgen.generate(YAML_PATH, config)
+
+        if os.path.isfile(SQL_PATH):
+            if os.path.isfile(JAVA_PATH):
+                identical_files = filecmp.cmp(SQL_PATH, JAVA_PATH)
+                self.assertFalse(identical_files, msg=f"""There's a SQL DDL file {JAVA_PATH} 
+                that's identical with the expected {SQL_PATH}""")
+            else:
+                print(f"{JAVA_PATH} is not accessible")
```
Comment (cmungall) [inline tests/test_issues/test_issue_571_sql_java.py:66]: avoid prints

**Comment by sierra-moxon** [comment]
@turbomam - ok to close this PR?  (opened last January)

**Pull request closed without merging**